doc/faq/howto_faq.rst

```diff
@@ -22,7 +22,7 @@ As of Matplotlib 2.2, `numpy.datetime64` objects are handled the same way
 as `datetime.datetime` objects.
 
 If you prefer the pandas converters and locators, you can register them.  This
-is done automatically when calling a pandas plot function and may be
+is done automatically when calling a pandas plot function and maybe
```

Comment:
```suggestion
is done automatically when calling a pandas plot function and may be
```

https://www.google.ca/amp/s/dictionary.cambridge.org/amp/british-grammar/maybe-or-may-be